driver/test/datetime_it.cpp

```diff
@@ -0,0 +1,235 @@
+#include "driver/platform/platform.h"
+#include "driver/test/client_utils.h"
+#include "driver/test/client_test_base.h"
+
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+
+#include <iostream>
+
+class DateTime
+    : public ClientTestWithParamBase<
+        std::tuple<
```

Comment:
IMO it is better to use named struct here, it is easier to read and reason about.